services/121-service/src/programs/dto/create-program.dto.ts

```diff
@@ -94,15 +94,18 @@ const exampleAttributes: ProgramRegistrationAttributeDto[] = [
 
 export class CreateProgramDto {
   @ApiProperty({ example: false })
+  @IsOptional()
   @IsBoolean()
   public readonly validation: boolean;
```

Comment:
Each `IsOptional` should change the relevant property to be optional like so:
```ts
  public readonly validation?: boolean;
```

Otherwise the frontend doesn't know it's optional.